tests/integration/test_mysql_protocol/test.py

```diff
@@ -836,33 +842,32 @@ def test_java_client(started_cluster, java_container):
     assert stdout.decode() == reference
 
 
-def test_prepared_statements(started_cluster, java_container):
-    with open(os.path.join(SCRIPT_DIR, "prepared_statements.reference")) as fp:
+def test_java_client_text(started_cluster, java_container):
```

Comment:
Now runs in both binary (server prepared statements) and text (regular queries) variants.